bugsnag/utils.py

```diff
@@ -118,7 +118,19 @@ def _sanitize_dict(self, obj, trim_strings, ignored):
 
         clean_dict = {}
         for key, value in six.iteritems(obj):
-            clean_dict[key] = self._sanitize(value, trim_strings, ignored)
+
+            clean_value = self._sanitize(value, trim_strings, ignored)
+
+            if isinstance(key, basestring):
```

Comment:
Our library is used in both Python 2.7 and 3, as such `six.string_types` is necessary here for compatibility.